docs/reference/TiffTags.rst

```diff
@@ -50,10 +52,11 @@ metadata tag numbers, names, and type information.
         >>> TAGS[0x010e]
         'ImageDescription'
 
-    This dictionary contains a superset of the tags in TAGS_V2, common
+    This dictionary contains a superset of the tags in :py:data:`~PIL.TiffTags.TAGS_V2`, common
     EXIF tags, and other well known metadata tags.
 
-.. py:attribute:: PIL.TiffTags.TYPES
+.. py:data:: PIL.TiffTags.TYPES
+    :type: dict
```

Comment:
Please compare:
* https://pillow--4697.org.readthedocs.build/en/4697/reference/TiffTags.html
* https://pillow.readthedocs.io/en/stable/reference/TiffTags.html

There's stuff missing from the new one.

Reply:
See other thread.